### DISCUSSION TITLE
How to make a database dump from Hasura Cloud to local Hasura (docker)?

### DISCUSSION THREAD

**Discussion by sarasusanv** [hasura/graphql-engine#7765] [Question]
Hello,

I have a free project on Hasura Cloud and I would like to dump it on Hasura installed on my computer (Docker).

I can't find any tutorial about it in the documentation.

Any advice?

Thanks!

**Comment by BenoitRanque** [comment]
Hasura cloud is a bring your own database solution. This means you would defer to whatever service is hosting your database for instructions on taking a backup. On hasura cloud this is often heroku.

One of the easiest way to take a database dump would be to install the PG admin desktop application, and connect to your database using your database connection string which you can find in your environment variables.

Please note that hasura cloud does not store your metadata or migration status in your database. You will have to restore those separately.
